projects/vdk-plugins/vdk-control-api-auth/src/vdk/plugin/control_api_auth/authentication.py

```diff
@@ -21,6 +27,7 @@ def __init__(
         auth_discovery_url: str = None,
         auth_type: str = None,
         cache_locally: bool = False,
+        possible_jwt_user_keys=None,
```

Comment:
I would remove unlesss there is a very definite use case for it today.
YAGNI.

Reply:
I still stand by my point here, but also happy to approve!

Reply:
In the spirit of simplicity and avoiding overengineering, your approach is perhaps cleaner. 

But given the context, when dealing with external systems or components that can have variable behavior (like different authentication providers with different JWT token structures), building in some level of flexibility is almost mandatory if you want to provide robust solution.